homeassistant/components/workday/config_flow.py

```diff
@@ -72,6 +72,12 @@ def add_province_to_schema(
 def validate_custom_dates(user_input: dict[str, Any]) -> None:
     """Validate custom dates for add/remove holidays."""
     for add_date in user_input[CONF_ADD_HOLIDAYS]:
+        if add_date.find(",") > 0:
+            dates = add_date.split(",", maxsplit=1)
+            for date in dates:
+                if dt_util.parse_date(date) is None:
+                    raise AddDateRangeError("Incorrect date in range")
+            continue
         if dt_util.parse_date(add_date) is None:
```

Comment:
```suggestion
        if (
            _is_valid_date_range(add_date, AddDateRangeError)
            and dt_util.parse_date(add_date) is None
        ):
```

And than call it like this